src/Tempest/Database/DatabaseConfig.php

```diff
@@ -7,15 +7,35 @@
 final class DatabaseConfig
 {
     public function __construct(
-        public readonly DatabaseDriver $driver,
-        public array $migrations = [],
+        private readonly DatabaseDriver $driver,
+        private array $migrations = [],
     ) {
     }
 
+    public function driver(): DatabaseDriver
+    {
+        return $this->driver;
+    }
+
     public function addMigration(string $className): self
     {
         $this->migrations[$className] = $className;
 
         return $this;
     }
+
+    public function set(array $migrations): void
+    {
+        $this->migrations = $migrations;
+    }
+
+    public function get(string $sort = 'asc'): array
```

Comment:
helper to get and sort the migrations.

Reply:
I guess you need to sort them both ways for up and down? I don't really like hard-coded strings. But it also feels weird to make an enum for it. A boolean also feels icky… Would it make sense to do the sorting in place (where the get method is called)? I reckon it's only one place, but I could be wrong…

Reply:
Also the naming is a bit weird, shouldn't it be `getMigrations`?

Reply:
>  I don't really like hard-coded strings. But it also feels weird to make an enum for it. A boolean also feels icky

I feel the same way 😅 this in combination with the match expression seemed like a good idea. But perhaps I'll move the sorting the the migration manager.